{"CVE Code": "CVE-2022-45693", "Description": "Jettison before v1.5.2 was discovered to contain a stack overflow via the map parameter. This vulnerability allows attackers to cause a Denial of Service (DoS) via a crafted string.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2022-45693", "Vulnerability Categories": "Memory Corruption;Denial of service", "Affected Products": "\nDebian\u00bbDebian Linux\u00bbVersion:10.0cpe:2.3:o:debian:debian_linux:10.0:*:*:*:*:*:*:*Matching versions\nDebian\u00bbDebian Linux\u00bbVersion:11.0cpe:2.3:o:debian:debian_linux:11.0:*:*:*:*:*:*:*Matching versions\nJettison Project\u00bbJettisonVersions  before (<) 1.5.2cpe:2.3:a:jettison_project:jettison:*:*:*:*:*:*:*:*Matching versions", "EPSS Score": "0.11% Probability of exploitation activity in the next 30 days\n~ 44 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n 1  2024-12-17  0.10%  0.11%  +0.00 \n 2  2024-08-03  0.10%  0.10%  +0.01 \n 3  2024-07-05  0.07%  0.10%  +0.03 \n 4  2024-05-25  0.06%  0.07%  +0.01 \n 5  2024-01-15  0.07%  0.06%  0.00 \n 6  2023-12-20  0.05%  0.07%  +0.01 \n 7  2023-11-13  0.05%  0.05%  +0.01 \n 8  2023-05-08  0.05%  0.05%  -0.01 \n 9  2023-04-17  0.06%  0.05%  0.00 \n 10  2023-03-22  0.05%  0.06%  +0.01 \n 11  2023-03-07  0.95%  0.05%  -0.91 \n 12  2023-01-12  0.89%  0.95%  +0.07 ", "CVSS Scores": "7.5", "CWE": [{"CWE_ID": "CWE-787 Out-of-bounds Write", "CWE_link": "/cwe-details/787/Out-of-bounds-Write.html"}], "Reference": [{"ref_link": "https://lists.debian.org/debian-lts-announce/2022/12/msg00045.html", "ref_desc": "[SECURITY] [DLA 3259-1] libjettison-java security updateMailing List;Third Party AdvisoryCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/3v6RtX8QFc4JvMXspDmBsu8PDIk.html"}, {"ref_link": "https://www.debian.org/security/2023/dsa-5312", "ref_desc": "Debian -- Security Information -- DSA-5312-1 libjettison-javaThird Party AdvisoryCVEs referencing this url", "ref_cve_link": "https://www.cvedetails.com/reference-url-info/ARG73o0wxasRD4Xr1ONcOtwAcAA.html"}, {"ref_link": "https://github.com/jettison-json/jettison/issues/52", "ref_desc": "If the value in map is the map's self, the new new JSONObject(map) cause StackOverflowError which may lead to dos \u00b7 Issue #52 \u00b7 jettison-json/jettison \u00b7 GitHubExploit;Issue Tracking;Third Party Advisory", "ref_cve_link": "N/A"}]}